pixi.toml

```diff
@@ -33,6 +32,7 @@ typos = ">=1.23.1,<2"
 [feature.lint.tasks]
 pre-commit-install = "pre-commit install"
 pre-commit-run = "pre-commit run --all"
+lint = {depends-on = ["pre-commit-run"]}
```

Comment:
```suggestion
lint = { depends-on = ["pre-commit-run"] }
```

Reply:
Should we maybe add a formatter for TOML to our CI pipeline?

Reply:
I would like that